activities/member-relations/founding-membership-workshop.md

```diff
@@ -0,0 +1,31 @@
+---
+type: Resource
+---
+
+This is a template agenda for workshops or discussions between potential founding members and the Foundation for Public Code staff.
+
+# Agenda
+
+Introduction round (10min)
+
+Background to the Foundation and current status (15min)
+
+Review membership proposal (30min)
+* Foundation governance structure, financial set up, workflow, 2019 roadmap and high-level overview of potential founding member's codebases for stewardship
+
+Break (10 min)
+
+Deep dive into Foundation processes (3x30min)
+* [Codebase stewardship process](../codebase-stewardship/index.md)
+* [Standard for Public Code](http://standard.publiccode.net/)
+* Foundation governance and financial model
+
+Break (10 min)
+
+Deep dive into founding member's codebases (45 min)
+* Based on proposed [codebase criteria](../codebase-stewardship/criteria-for-codebase-stewardship.md)
+
+Recap and next steps (10 min)
+
+## Preparation
+```Insert custom preparation notes here```
```

Comment:
Can we remove this and this code block, there is no code here.

Reply:
Also the header would be left as an abandonned parent, it should also be removed.